tests/responses/tests.py

```diff
@@ -136,6 +138,20 @@ def test_repr(self):
         expected = '<HttpResponse status_code=201, "text/html; charset=utf-8">'
         self.assertEqual(repr(response), expected)
 
+        response = HttpResponse(status=204)
+        del response['Content-Type']
+        expected = '<HttpResponse status_code=204>'
+        self.assertEqual(repr(response), expected)
```

Comment:
Tests should generally follow the Arrange, Act, Assert pattern, i.e. you have 4 tests here so they should be in four methods, e.g. `test_repr_content_type`, `test_repr_no_content_type`, etc. See http://jamescooke.info/cleaner-unit-testing-with-the-arrange-act-assert-pattern.html